### PR TITLE
(DOCSP-33623): Broke out search tutorials.

### PR DESCRIPTION
**Pull request by corryroot**
@boooczek and @lmkerbey-mdb, I broke the search sections into their own tutorial:

[STAGE: Deploy tutorials landing](https://preview-mongodbcorryroot.gatsbyjs.io/atlas-cli/DOCSP-33623/atlas-cli-local-cloud/)

[STAGE: Local deploy tutorial](https://preview-mongodbcorryroot.gatsbyjs.io/atlas-cli/DOCSP-33623/atlas-cli-deploy-local/)

[STAGE: Search tutorial](https://preview-mongodbcorryroot.gatsbyjs.io/atlas-cli/DOCSP-33623/atlas-cli-deploy-fts/)

[JIRA](https://jira.mongodb.org/browse/DOCSP-33623)

[BUILD](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=654a42cf5b686d6bd90189a9)

### CODE REVIEW

**Review by lmkerbey-mdb — APPROVED** [review]
LGTM.

**Review by boooczek — CHANGES_REQUESTED** [review]

File: source/atlas-cli-deploy-fts.txt [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+        curl  https://atlas-education.s3.amazonaws.com/sampledata.archive -o sampledata.archive
+
+  #. Copy and paste the following command into your terminal and 
+     replace ``{port-number}`` with the port for your 
+     deployment:
+
+     .. code-block:: sh
+
+        mongorestore --archive=sampledata.archive --port={port-number}
+
```
[the remark this answers is not in the document]
Comment (corryroot) [reply source/atlas-cli-deploy-fts.txt:49]: Added:

```
- Install :dbtools:`MongoDB Database Tools 	
  </installation/installation/>`.	

  **Example:**	

  .. code-block:: sh	

     brew install mongodb-database-tools
```

File: source/atlas-cli-deploy-fts.txt [hunk 2]
```diff
@@ -0,0 +1,529 @@
...
+   .. step:: Run an example query.
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: sh
+            
+            db.embedded_movies.aggregate([
```
Comment (blva) [inline source/atlas-cli-deploy-fts.txt:449]: how are we wording this tutorial in other docs? I am not able to copy this command into the shell and I remember this was an issue that mongosh wanted to fix. as of now mongosh gets stuck for me

Comment (corryroot) [reply source/atlas-cli-deploy-fts.txt:449]: @blva, it does take a while to copy and paste such a long vector query into the command line. It should paste over eventually. @kanchana-mongodb, does the Search team have any thoughts on this?

Comment (kanchana-mongodb) [reply source/atlas-cli-deploy-fts.txt:449]: In my testing also, it takes a while to copy/paste, but has never gotten stuck. I can ask the search team about this, but doubt there's much they can do about the delay in being able to paste the query into the terminal.

Comment (blva) [reply source/atlas-cli-deploy-fts.txt:449]: cool! great to know. should we add a warning/comment for customers to ask them to wait? would be nice to have a time estimate as well

Comment (corryroot) [reply source/atlas-cli-deploy-fts.txt:449]: @blva, sure I'll add an important note that copying and pasting a large query vector might take a while depending on your machine.

**Review by blva — APPROVED** [review]

**Pull request merged**